src/elm/Pages/Build/View.elm

```diff
@@ -796,9 +774,9 @@ expandAllButton expandAll org repo buildNumber =
 {-| logsHeader : takes number, filename and decoded log and renders logs header
 -}
 logsHeader : LogsMsgs msg -> String -> String -> String -> String -> Html msg
-logsHeader msgs resource number fileName decodedLog =
+logsHeader msgs resource number fileName logData =
     div [ class "logs-header", class "buttons", Util.testAttribute <| "logs-header-actions-" ++ number ]
-        [ downloadLogsButton msgs.download resource number fileName decodedLog ]
+        [ downloadLogsButton msgs.download resource number fileName logData ]
```

Comment:
changed this to use the raw data, offsetting the base64 decode work to the actual button click.